admin/workspace-providers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Coder allows you to add to your deployment multiple Kubernetes clusters. These
+additional clusters are called **providers**. This can be helpful for
+distributed teams, since users can choose the provider closest to them to
+minimize latency.
+
+## Enabling Workspace Providers
+
+To enable the Workspace Providers UI elements:
+
+1. In the Coder UI, click on your avatar.
+2. In the menu that appears, click **Feature Preview**.
+3. Select **Resource Pools UI** and click **Enable**.
+
+Once enabled, this allows end-users to choose the workspace provider and
+namespace where they want their environment deployed.
+
+## Admin UI
+
+Site managers and admins can view the workspace providers configuruation
+page avaiable via **Manage** > **Admin** > **Workspace Providers**.
+
+![Workspace Providers Admin](../assets/workspace-providers-admin.png)
+
+The Admin panel will provide high-level information on all of your workspace
+providers, as well as whether the provider is active on not. The default tag
+indicates the provider that will be automatically selected whenever a user
+launches the Create an Environment modal.
```

Comment:
Yeah -- and the flag to enable it is "Resource Pools UI" which confused me a bit too. I think we need to clean this up for 2.0 since I don't think these will make it this time around